unicodetools/src/main/java/org/unicode/text/UCD/UCD_Names.java

```diff
@@ -414,7 +414,7 @@ public final class UCD_Names implements UCD_Types {
         // Unicode 15
         "Kawi",
         "Nag_Mundari",
-        // A future version of Unicode
+        // Unicode 16
         "Sunuwar",
         "Tulu_Tigalari",
         "Kirat_Rai",
```

Comment:
FYI -- not really important / nice-to-have

For the last several versions, we have sorted scripts alphabetically within each version.

Reply:
Alphabetically by script name or by script code?

Reply:
Good question. Judging from the Unicode 11 list, it's by script _code_, at least at that time.
```
        "Sogdian",
        "Old_Sogdian",
    ...
        "Sogd",
        "Sogo",
```